hw/ip/pinmux/doc/checklist.md

```diff
@@ -18,7 +18,7 @@ RTL           | [IP_TOP][]                     | Done        |
 RTL           | [IP_INSTANTIABLE][]            | Done        |
 RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Done        |
 RTL           | [FUNC_IMPLEMENTED][]           | Done        |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Done        |
+RTL           | [ASSERT_KNOWN_ADDED][]         | Done        | Primary I/Os are exempted from KNOWN assertions since the chip-level testbench may drive X'es onto some of these signals.g
```

Comment:
```suggestion
RTL           | [ASSERT_KNOWN_ADDED][]         | Done        | Primary I/Os are exempted from KNOWN assertions since the chip-level testbench may drive X'es onto some of these signals.
```